bpm/bonita-core/bonita-process-instance/bonita-process-instance-impl/src/main/resources/org/bonitasoft/engine/core/process/instance/model/impl/hibernate/process.instance.queries.hbm.xml

```diff
@@ -83,6 +83,7 @@
 		WHERE g.logicalGroup4 = :parentProcessInstanceId
 		AND g.name = :name
 		AND g.terminal = FALSE
+		AND g.hitBys NOT LIKE 'FINISH%'
```

Comment:
Yep but I was refering to the explain, in this case it does a *Seq Scan on flownode_instance* and with or without the "not like" it does not change the explain at all. So I was wondering if the explain can change depending on the database size.
In any case even without the "not like" no index is used